### DISCUSSION TITLE
Documenting example layouts

### DISCUSSION THREAD

**Discussion by elParaguayo** [qtile/qtile#3457] [Show and tell]
A quick show and tell.

Having added widget screenshots to the docs, I'm now working on layouts. Some quick examples:
![2](https://user-images.githubusercontent.com/946265/160483703-bb86cca2-e29d-49ad-86a6-c191aa7846ef.gif)

![2](https://user-images.githubusercontent.com/946265/160483730-b67a45fb-7ac5-4088-b5f4-1aabb041b928.gif)

![2](https://user-images.githubusercontent.com/946265/160483752-979becc1-3efc-4567-8466-f738f92ee259.gif)

Colour scheme is not final!

**Comment by thingsiplay** [comment]
Seeing this animated is a great idea and helps illustrating it in seconds without words. It is a bit more compact than the current illustrations, if we take the multiple states into account. About the coloring, it is not bad, but would prefer a less prominent one like the current green background for the "ascii"-illustrations.

Edit: I don't know if this goes too far and how easy it is to do, but is it possible to play animation only on hover or pause the animation when mouse is hover the image?

**Reply by elParaguayo** [reply]
Hi. Thanks for the comments.

So, the current colouring looks like this:
![3](https://user-images.githubusercontent.com/946265/162559965-5e1489b2-7b76-4c22-8a5e-c9d8c5c25f17.gif)

This is consistent with the theme in the docs. Also, ignore the weird text colours - that's fixed now!

As for your second question, I think the answer is "no". It probably *is* possible with css/javascript but that's a bit more complicated when using sphinx for building our docs so I don't plan on doing it.

**Reply by thingsiplay** [reply]
Looks better. I noticed you added the border for active window or for the master stack. It is easy to miss, due to the red on blue and small size. My next suggestion is to increase the size of border for active/master window. Also the animation is a bit too fast, for my taste. This is really just my opinion, so anyone else feel free to comment on this.

**Reply by SweepingBishops** [reply]
A tad bit slower would be better. As for the borders, I prefer them as it is now. Though increasing the thickness would make it easier to notice for people who don't specifically look for it.

**Reply by elParaguayo** [reply]
Changing speed is easy.

As for borders, it's important to show default values but I appreciate that this makes things hard to see when the image is scaled down. 

I can add examples with a border setting.

**Comment by hamensman** [comment]
Very pedantic, but the new_client position `MonadThreeCol` here is not consistent with the other two - (i.e. master should still be 1)

**Reply by elParaguayo** [reply]
Fair point but that's an issue with the layout, not the screenshots!
